service/step_defs_test.go

```diff
@@ -1527,7 +1527,7 @@ func (f *feature) aValidControllerGetCapabilitiesResponseIsReturned() error {
 			}
 		}
 
-		if count != 10 {
+		if count != 9 {
```

Comment:
i guess there are 10 capabilities, the count should be 10

Reply:
we removed the list_volumes one, because then health sidecar would use that one instead of list_volumes.